src/components/Header.js

```diff
@@ -17,11 +18,17 @@ export default (props) => {
     window.location.href = config.TERM_URL
   }
 
-  let goUserHome = (userID) => {
+  let goUserHome = () => {
+    window.location.href = '/user/' + userID
+  }
+
+  let renderUserHome = () => {
     if(!userID || userID === PTT_GUEST) {
-      return
+      return <Empty />
```

Comment:
We would like to change to `<a className={'pull-right' + styles['navbar-link']} href="/login">hi~guest</a>`

Reply:
Got it~~